### PR TITLE
fix(#31): fixes #31 issue with null duration

### LINKED ISSUE #31: Skipped test suite results in broken output
Test suite skipped at too level, like this:

```js
describe.skip('Completely skipped suite', () => {
  it('some test', () => {})
})
```
results in output where each of the internal tests is listed with `null` at duration field, as a result SonarScannel throws on that file as being invalid.

### PR DESCRIPTION
**Pull request by jlestes**
When a test is skipped or marked as a todo the duration is reported
as null. This fix defaults a nullish value to 0 to prevent errors
from SonarQube.

### CODE REVIEW

File: __test__/reporter/absolute-path-reporter.spec.js [hunk 1]
```diff
@@ -66,7 +66,7 @@ describe('AbsolutePathReporter', () => {
                         testResults: [
                             {
                                 fullName: 'Skipped "this test is skipped"',
-                                duration: 10,
```
Comment (sh33dafi) [inline __test__/reporter/absolute-path-reporter.spec.js:69]: Add an additional testresultaten to test the null case, since we still need a test to cover the other case

**Review by sh33dafi — CHANGES_REQUESTED** [review]

File: __test__/reporter/absolute-path-reporter.spec.js (hunk 1 above)
Comment (jlestes) [reply __test__/reporter/absolute-path-reporter.spec.js]: @sh33dafi addressed the feedback and added a new test case.
